scipy/stats/_continuous_distns.py

```diff
@@ -8565,20 +8565,19 @@ def _log_sum(log_p, log_q):
 
 # same as above, but using -exp(x) = exp(x + πi)
 def _log_diff(log_p, log_q):
-    # need to broadcast in case a or b is 0.5; logsumexp doesn't
+    # need to broadcast in case a or b is 0; logsumexp doesn't
     log_p, log_q = np.broadcast_arrays(log_p, log_q)
```

Comment:
This is probably not needed anymore due to the change in `mass_case_central`, but I'll remove it later.